anitya/mail_logging.py

```diff
@@ -34,7 +34,9 @@
 
 psutil = None
 try:
-    import psutil
+    import psutil as _psutil
```

Comment:
Why to do it like this?

Reply:
Because the linter complained about psutil being shadowed by the import.

Reply:
Ok, in this case fine